server/handlers/connections_handlers.go

```diff
@@ -13,6 +13,8 @@ import (
 	"github.com/layer5io/meshkit/models/events"
 )
 
+type connectionStatusPayload map[uuid.UUID]models.ConnectionStatus
```

Comment:
@MUzairS15, does this map to the Status attribute in the Connections OpenAPI schema (are we being schema-driven)?